### PR TITLE
Some improve for docs

### PR DESCRIPTION
**Pull request by Yushu2606**
规范LLSE为脚本引擎/脚本插件
规范DLL插件/LL插件为原生插件（/.NET插件）
规范C++后缀名为`.cpp`和`.h`
规范其他项目名称（如Js->JavaScript/JS，Json->JSON等）
规范LiteLoaderBDS与LiteLoader
移除部分会引起歧义的缩写
修改部分语句

### CODE REVIEW

**Comment by futrime** [comment]
Please describe what you have done in detail.

**Comment by StarsDream00** [comment]
> Please describe what you have done in detail.

it is writing

File: en/DotNETPluginDevelopment/APIs/Namespace/LLNET.AllowList/Class/AllowListManager/AllowListManager.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <br>
 
 C#
-```cs
+```csharp
```
Comment (futrime) [inline en/DotNETPluginDevelopment/APIs/Namespace/LLNET.AllowList/Class/AllowListManager/AllowListManager.md]: Please do not modify the language code, which conforms to the specifications of Prism.js and keeps a simple form.

File: en/LLSEPluginDevelopment/GameAPI/Command.md [hunk 2]
```diff
@@ -12,10 +12,12 @@ The following APIs provide interfaces for registering and listening to custom co
 - Return value: Whether the execution was successful.
 - Return value type:  `Boolean`
 
-```Javascript
-[JS]
+[JavaScript]
```
Comment (futrime) [inline en/LLSEPluginDevelopment/GameAPI/Command.md:15]: Not beautiful.

File: en/Maintenance/StyleGuide.md [hunk 3]
```diff
@@ -4,15 +4,15 @@
 
 ## 头文件
 
-一般来说，每个 `.cpp` 文件都应该有一个同名的的 `.h` 文件。有一些常见的例外情况，如单元测试和只包含 `main()` 函数的小型 `.cpp` 文件。
+一般来说，每个 `.cpp` 文件都应该有一个同名的的 `.hpp` 文件。有一些常见的例外情况，如单元测试和只包含 `main()` 函数的小型 `.cpp` 文件。
```
Comment (futrime) [inline en/Maintenance/StyleGuide.md]: DO NOT modify the style guide, which has been discussed by us LiteLDev, considering both the Google C++ Style Guide and the code already in LiteLoaderBDS.

File: en/Usage.md [hunk 4]
```diff
@@ -59,11 +59,13 @@ Now that you have LiteLoaderBDS installed, how about adding some plugins?
 
 ## 🎯 Add Some Plugins
 
-There are two types of plugins: LL plugins and LLSE plugins.
+There are 3 types of plugins: Native plugins, Script plugins and .NET plugins.
```
Comment (futrime) [inline en/Usage.md]: The .NET plugin belongs to the native plugin, also called LL plugin.

File: zh-Hans/_sidebar.md [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 - [❓ 常见问题](/FAQ.md)
 
-- [⛳ C++插件开发](https://cpp.docs.litebds.com/zh-Hans)
+- [⛳ 原生插件开发](https://cpp.docs.litebds.com/zh-Hans)
```
Comment (futrime) [inline zh-Hans/_sidebar.md]: The C++ plugin does not equal to the native plugin.

**Review by futrime — CHANGES_REQUESTED** [review]
Thank you for your PR. Please make changes referring to my comments. DO NOT change the `cs`, `js`, `cpp` and `lua` language tags.

File: en/Usage.md (hunk 4 above)
Comment (Yushu2606) [reply en/Usage.md]: .NET插件是由LL.NET加载的，运行机制及特性同步均与原生插件不同

Comment (Yushu2606) [reply en/Usage.md]: 并且LL缩写本身就带有歧义（整个 LiteLoaderBDS项目/LiteLoader原生插件加载部分/LiteLoader加载器公用API）

File: en/Maintenance/StyleGuide.md (hunk 3 above)
Comment (Yushu2606) [reply en/Maintenance/StyleGuide.md]: 现有的LiteLoaderBDS项目本就使用了.cpp/.hpp后缀，文档应以当前项目为准实时修改

Comment (futrime) [reply en/Maintenance/StyleGuide.md]: The maintenance documentation is written for further development, including refactoring.

File: en/Usage.md (hunk 4 above)
Comment (futrime) [reply en/Usage.md]: Thanks for your hints. I feel quite sorry that not informing that .NET support is just under development, which should not be used in production environment and plugin development. I'm hiding these pages from the sidebar.

Comment (futrime) [reply en/Usage.md]: Please remove the .NET plugin item.

File: zh-Hans/LLSEPluginDevelopment/DataAPI/PermAPI.md [hunk 6]
```diff
@@ -7,7 +7,7 @@
 整个系统可以被分成三部分：`Role`(身份组), `Permission`(`PermInstance`, 权限实例) 和 `PermInfo`(权限信息)。  
 `Role`定义了一组拥有指定权限的玩家，`PermInfo`则储存了所有权限的描述。  
 
-和Discord一样，我们也有默认的`admin`(管理员)和`everyone`(所有人)身份组。    
+和`Discord`一样，我们也有默认的`admin`(管理员)和`everyone`(所有人)身份组。
```
Comment (futrime) [inline zh-Hans/LLSEPluginDevelopment/DataAPI/PermAPI.md]: 没必要改

File: zh-Hans/LLSEPluginDevelopment/EventAPI/EntityEvents.md [hunk 7]
```diff
@@ -43,8 +43,8 @@ DamageCause 为伤害原因枚举，枚举值如下，有问号的待验证，
 | Projectile | 0x03 | 抛射物攻击 |
 | Suffocation | 0x04 | 窒息（密封空间） |
 | Fall | 0x05 | 掉落 |
-| Fire | 0x06 | 着火 |
-| FireTick | 0x07 | 着火 |
+| Fire | 0x06 | 在火中烧死 |
```
Comment (futrime) [inline zh-Hans/LLSEPluginDevelopment/EventAPI/EntityEvents.md]: Not died yet.

File: zh-Hans/LLSEPluginDevelopment/EventAPI/PlayerEvents.md [hunk 8]
```diff
@@ -18,7 +18,7 @@
     正在连接服务器的玩家对象
 - 拦截事件：函数返回`false`
```
Comment (futrime) [inline zh-Hans/LLSEPluginDevelopment/EventAPI/PlayerEvents.md:20]: XUID

**Review by futrime — CHANGES_REQUESTED** [review]

**Review by futrime — APPROVED** [review]

**Pull request merged**